app/views/layouts/_header.html.erb

```diff
@@ -17,8 +17,8 @@
     <!-- Collect the nav links, forms, and other content for toggling -->
     <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
       <ul class="nav navbar-nav">
-        <li class="active"><a href="/">Home<span class="sr-only">(current)</span></a></li>
-        <li><a href="#">Popular</a></li>
+        <li><%= active_link_to 'Home', root_path, wrap_tag: 'li', active: :exclusive %></li>
+        <li><%= active_link_to 'Popular', popular_products_path, wrap_tag: 'li', active: :exclusive %></li>
```

Comment:
the `wrap_tag` is interesting; is this just telling the link that it's *already* wrapped by an `<li>` tag, or is it explicitly wrapping the link in an `<li>` tag (which would result in duplicated list tags `<li><li>`, I think)?